Source/WebInspectorUI/UserInterface/Controllers/JavaScriptLogViewController.js

```diff
@@ -367,6 +379,21 @@ WI.JavaScriptLogViewController = class JavaScriptLogViewController extends WI.Ob
         if (this.delegate && typeof this.delegate.didAppendConsoleMessageView === "function")
             this.delegate.didAppendConsoleMessageView(messageView);
     }
+
+    _handleShowConsoleMessageTimestampsSettingChanged()
+    {
+        this._showTimestamps = WI.settings.showConsoleMessageTimestamps.value;
+        if (this._showTimestamps) {
+            this._currentSessionOrGroup.element.removeAttribute("hide-timestamps");
+            for (let i = 0; i < this._consoleMessageViews.length; i++) {
+                let consoleMessageView = this._consoleMessageViews[i];
+                if (consoleMessageView.constructor.name == "ConsoleMessageView")
```

Comment:
Can views other than `WI.ConsoleMessageView`s end up in this array? If not, there isn't a need to check this.

Reply:
Yes other views have consistently ended up in this array

Reply:
👍 Sound good

Should have also mentioned yesterday that if keep the check we tend to prefer 
```
consoleMessageView instanceof WI.ConsoleMessageView
```
for type checks.